plugins/woocommerce/src/Blocks/Utils/BlockHooksTrait.php

```diff
@@ -38,18 +38,9 @@ public function register_hooked_block( $hooked_blocks, $position, $anchor_block,
 		 *
 		 * @since $VID:$
 		 */
-		$pattern_exclude_list = apply_filters( 'woocommerce_hooked_blocks_pattern_exclude_list', array( 'twentytwentytwo/header-centered-logo', 'twentytwentytwo/header-stacked' ) );
+		$pattern_exclude_list = apply_filters( 'woocommerce_hooked_blocks_pattern_exclude_list', array( 'twentytwentytwo/header-centered-logo', 'twentytwentytwo/header-stacked', 'blockbase/header-centered' ) );
```

Comment:
I think we should do both. Exclude it for now > Contribute > Include it back in. I'm leaving it excluded for now because if we want to fix this issue it'll fall quite low on the priority list for now and I'm unsure on timescales.

Reply:
Makes sense to me 🤝